tooling/template-base/atoms/AudioPlayerBar.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect, useState } from 'react';
+
+export default function TopicAudioPlayer({ keyName }: { keyName: string }) {
+  return <AudioP keyName={keyName} />;
+}
+
+function AudioP({ keyName }: { keyName: string }) {
```

Comment:
There's no need for `AudioP` here, the function definition can just be inline with `TopicAudioPlayer`. (this was just me debugging some React behavior I was seeing)